WalletWasabi.Gui/CommandLine/MixerCommand.cs

```diff
@@ -25,7 +25,7 @@ public MixerCommand()
 				{ "h|help", "Displays help page and exit.",
 					x => ShowHelp = x != null},
 				{ "w|wallet=", "The name of the wallet file.",
-					x =>  WalletName = x?.ToLower() },
+					x =>  WalletName = x },
```

Comment:
This'd make wallet names case sensitive only on Linux.

Reply:
ack